tools/encodings/arabic_unique-glyphs.nam

```diff
@@ -79,74 +79,29 @@
 0x0670 # ARABIC LETTER SUPERSCRIPT ALEF
 0x0671 # ARABIC LETTER ALEF WASLA
 0x0679 # ARABIC LETTER TTEH
-0x067A # ARABIC LETTER TTEHEH
```

Comment:
It seems that on investigating this a bit more, the larger set that has been here was stale, and this PR simply updates the file to reflect what has been in use since march.